server/data/quotes.json

```
@@ -23,6 +23,10 @@
     "author": "Amy Chua",
     "sentence": "Do you know what a foreign accent is? It’s a sign of bravery."
   },
+  {
+    "author": "Amy Santiago",
+    "sentence":"Some Say latin is a dead language but I say its just taking a long nap." 
```

Thanks for the PR. Please update it as shown:

```suggestion
    "sentence": "Some Say latin is a dead language but I say its just taking a long nap." 
```